### PR TITLE
Add usage info for omit-mount-info

### PR DESCRIPTION
**Pull request by Haoning-Sun**
### What changes are proposed in this pull request?

Add usage info for omit-mount-info.

### Why are the changes needed?
Show usage for omit-mount-info parameter.

### CODE REVIEW

**Comment by ChunxuTang** [comment]
Hi @Haoning-Sun, would your PR need some updates on the Alluxio documentation?

**Comment by Haoning-Sun** [comment]
@ChunxuTang I didn't find the document that needs to be updated, do you have any suggestions?

**Comment by LuQQiu** [comment]
https://docs.alluxio.io/os/user/stable/en/operation/User-CLI.html#ls

whose source code is https://docs.alluxio.io/os/user/stable/en/operation/User-CLI.html#ls

**Comment by Haoning-Sun** [comment]
@LuQQiu @ChunxuTang Thanks. I will add it.

File: docs/en/operation/User-CLI.md [hunk 1]
```diff
@@ -1203,6 +1203,7 @@ By default, it loads metadata only at the first time at which a directory is lis
 * `-r` reverses the sorting order.
 * `--timestamp` display the timestamp of the given option. Possible values are creationTime, lastModificationTime, and lastAccessTime.
 The default option is lastModificationTime.
+* `-m` option exclude mount point related information.
```
Comment (ChunxuTang) [inline docs/en/operation/User-CLI.md:1206]: typo: exclude => excludes

**Review by ChunxuTang — COMMENTED** [review]
LGTM. Just left a suggestion to fix a typo.

**Review by ChunxuTang — APPROVED** [review]
LGTM. Thanks for the work.

**Review by beinan — APPROVED** [review]

**Comment by beinan** [comment]
alluxio-bot, merge this please.

**Pull request merged**